src/main/java/io/chazza/advancementapi/AdvancementAPI.java

```diff
@@ -238,14 +252,13 @@ public String toString() {
         }
     }
 
-
     //BEGIN CLASSES
     @Builder(builderMethodName = "hiddenbuilder")
```

Comment:
I’ll create a process to automatically create a delomboked sourcefile from a Lomboked file.
So it’ll kept compact for development but allows the user to have a delomboked file.